packages/gatsby-cli/package.json

```diff
@@ -37,7 +37,7 @@
     "pretty-error": "^2.1.1",
     "progress": "^2.0.3",
     "prompts": "^2.3.2",
-    "react": "^16.8.0",
+    "react": "^16.12.0",
```

Comment:
Is this needed in this PR? https://github.com/gatsbyjs/gatsby/pull/21522 has some info why this is intentionally lowest it can be